WeScanTests/UIImageTests.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  UIImageTests.swift
+//  WeScanTests
+//
+//  Created by James Campbell on 8/8/18.
+//  Copyright © 2018 WeTransfer. All rights reserved.
+//
+
+import XCTest
+@testable import WeScan
+
+class UIImageTests: XCTestCase {
+        
+    func testScailingNonCGBackedUIImageReturnNil() {
```

Comment:
Little typo I believe: `testScailing...` vs `testScaling...`
This is true for the following functions as well.

Reply:
Cool submitting a fix for this too